diesel/src/macros/insertable.rs

```diff
@@ -199,6 +199,24 @@ macro_rules! _Insertable {
                 ,)+)
             }
         }
+
+    } __diesel_parse_as_item! {
+        impl<$($lifetime,)* 'insert, Op, Ret> $crate::query_builder::insert_statement::IntoInsertStatement<$table_name::table, Op, Ret>
+            for &'insert $struct_ty where
+                $('insert: $lifetime,)*
+                $crate::query_builder::insert_statement::InsertStatement<$table_name::table, &'insert $struct_ty, Op, Ret>: $crate::query_builder::AsQuery,
```

Comment:
Uhm, no? `'a: 'b` means that `'a` lives at least as long as `'b`. IIRC, this not really intuitive. But [the nomicon](https://doc.rust-lang.org/nomicon/subtyping.html) and [Niko](http://smallcultfollowing.com/babysteps/blog/2016/05/09/non-lexical-lifetimes-adding-the-outlives-relation/#the-outlives-relation-in-light-of-control-flow) seem to agree with me.